src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenLocalFunctionTests.cs

```diff
@@ -6164,12 +6164,6 @@ static void local([Attr] in State state)
             var param = localFunction.Parameters[0];
             Assert.True(param.IsMetadataIn);
             Assert.False(param.IsMetadataOut);
-
-            // Test a scenario where the baseParameterAttributes has a different RefKind than the synthesized parameter.
-            // We expect the RefKind of the base parameter to be ignored here.
-            var synthesizedParam = SynthesizedParameterSymbol.Create(localFunction, param.TypeWithAnnotations, ordinal: 0, RefKind.Out, param.Name, baseParameterForAttributes: (SourceComplexParameterSymbolBase)param);
-            Assert.False(synthesizedParam.IsMetadataIn);
-            Assert.True(synthesizedParam.IsMetadataOut);
         }
```

Comment:
Seems like this deleted block of code was important to the test - at least from the name `BaseParameterWithDifferentRefKind` and the linked issue.

Reply:
Removed the test as we never need to synthesize a parameter where the refkind and the base parameter's refkind disagree.
The linked issue describes the scenario covered by `InOutAttributes`.
Tagging @RikkiGibson to confirm